src/main/java/nl/tudelft/jpacman/npc/ghost/Inky.java

```diff
@@ -56,24 +52,48 @@
 	 */
 	private static final int MOVE_INTERVAL = 250;
 
+	
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'chemin' could be replaced by a local variable.](https://www.codacy.com/app/mathieu-craeye/jpacman-framework_2/file/2532229389/issues/source?bid=3267846&fileBranchId=3284210#l56)